src/components/plots/PlotContainer.jsx

```diff
@@ -36,42 +52,51 @@ const PlotContainer = (props) => {
       return currentConfig[key] === initialConfig[key];
     });
 
-    return isConfigEqual;
+    return isEqual;
+  };
+
+  const handleResize = () => {
+    const direction = window.innerWidth > 1024 ? 'row' : 'column';
+    if (tileDirection !== direction) setTileDirection(direction);
   };
 
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
+  }, []);
+
   useEffect(() => {
     if (!config) {
       return;
     }
 
-    if (_.isEqualWith(config, initialPlotConfigStates[plotType], checkConfigEquality)) {
-      setEnableReset(false);
+    if (isConfigEqual(config, initialPlotConfigStates[plotType])) {
+      setResetDisabled(true);
       return;
     }
 
-    setEnableReset(true);
+    setResetDisabled(false);
   }, [config]);
 
   const onClickReset = (event) => {
     event.stopPropagation();
```

Comment:
can you check if we need to stop event propagation here? I think it might not be needed?

Reply:
Yep, it's not needed. Removed. I think this was also a carry over from a previous PR,